### DISCUSSION TITLE
Acustica space control

### DISCUSSION THREAD

**Discussion by fojerhar** [apohl79/audiogridder#952] [General]
Hi Andreas

There is an issue with the Acustica new space control plugin. It has only au and vst3 version yet, so I can only use it trough AG.
Beta9 works really nice btw.
I can open this plug, I can work with it, loads fast, has a small delay only, until this point is nice (and sounds great)
But as I save it with the session at the next opening pt quits and throws and error log. If I open a session where this is bypassed (I use it serial in a different ag plugin on the masterbus) I can activate it and goes well, I just have to bypass it before saving the session.
I wrote a ticket to the factory side, but they were not too cooperative.....
If you could demo this and take a look what is happening, that would be nice

still thank you for your amazing invention

**Comment by fojerhar** [comment]
And another issue, after saving closing and reopening a session the automation values disappearing, renaming to unused, while in the automation window still looks fine
![Screen Shot 2022-05-19 at 13 08 02](https://user-images.githubusercontent.com/72139943/169279911-529fa0c3-fe22-4156-b2f7-1460ede52175.png)
e

**Reply by apohl79** [reply]
Automation parameter naming is very much depending on the DAW. When the AG plugin loads, it will report unknown to the DAW for all parameters as no remote plugin is loaded at that time. Later after loading, the names can change, but the DAW has to read those changes. AG behaves the same way on all DAWs but each DAW is different.

**Comment by fojerhar** [comment]
Okay, I have to correct myself, with beta7 I mostly can reopen the sessions, having the issue with beta 8 and 9 (the spacecontrol)

**Reply by apohl79** [reply]
Even though the DAW does not matter, I tried logic and pro tools with the trial of spacecontrol. I can change it, save and reload the session and get the spacecontrol settings back.

Can you try the plugin isolation sandbox mode and see if that works for you?

**Reply by fojerhar** [reply]
Tried it
beta7 chain iso PT not responding
beta7 plain iso PT unexpectedly quit
beta9 chain iso PT not responding
beta9 plugin iso PT unexpectedly quit

beta9 chain iso and then activate this plug not responding
Stayed at beta7, in that I can enable the inactivated ag what only contains this plug
**It is not your fault, everything else is working fine.**
In smaller sessions where I have 8 chains with  50 instances that can be opened with beta7
Bigger ones with 12 workers 70+ plugs has issues

Weird:As session opens your chains are loading up properly. As session is loaded sandboxes collapsing and reloading with new times
![Screen Shot 2022-05-20 at 15 26 24](https://user-images.githubusercontent.com/72139943/169539230-925f310c-6d29-4cd8-bd2d-d445caab4d99.png)

And the big delay numbers are coming pt will crash. Reaper could wait at least 30-40 seconds to align the track properly, pt not.... immediately starts spinning the beachball(not responding)

Maybe their next update will be fine, until that I can use this way.

As I tried plugin isolation it caused higher cpu usage and and much more stops in the daw, that "audio processing deadline wan not met"
![Screen Shot 2022-05-20 at 15 40 18](https://user-images.githubusercontent.com/72139943/169540499-e2fe4f5d-d0a0-4133-86a1-02201c975b9f.png)

thank you for doublechecking it!

**Reply by apohl79** [reply]
> PT unexpectedly quit

This looks like the AG plugin is crashing PT. The problem is, PT has its own crash reporting. So there is no way I can catch a backtrace or something. Have you seen Reaper quitting as well? If yes I might find some info about it.

The response time in the plugin mon looks crazy. I have seen that spacecontrol is quite slow and easily uses >10ms on the server side, but 400+ does not look right. I guess thats because of crashes.

Please share the plugin and server logs. It might reveal something.

**Reply by fojerhar** [reply]
I think you cannot crash reaper.... :)
here are my logs and reports
https://www.dropbox.com/sh/2ihylrd22wgg9el/AAA3UAScCRAAdVmczMe_GeA7a?dl=0

At first I opened a session what contains SPC and could be opened at anytime
then opened a bigger one then activated SPC, saved and reopened and had a crash 
after this, the server collapsed and had to restart the computer
beta7 till this point
after restart opened the new saved session and it opened quickly, in this point I realized my sonar works systemwide was not loaded.
Without it I had more success opening sessions, so that also makes a trouble with audio clocking and timing 
than beta9
60-60% of successes
here is a frozen PT sample from activity monitor, maybe you can read something from it
[Sample of Pro Tools2.txt](https://github.com/apohl79/audiogridder/files/8747317/Sample.of.Pro.Tools2.txt)

now back to beta7, without sonarworks systemwide it opens the session , and now the sonarworks plugin causes this hangs I think

I m not totally sure, cannot reproduce the same result all the time, sometimes opens quickly and goes well, sometimes hangs out or quits unexpectedly

But having trouble only with this combo: spacecontrol and sonarworks

**Reply by fojerhar** [reply]
Hi
Have to say, that the problem disappeared for this week
with beta9 session opens well, I don t know what changed....

and we bought the M1 studio ultra, and there beta9 works also really well with protools... :)
